part-2-server/node_poe/server.js

```diff
@@ -0,0 +1,25 @@
+var http = require('http');
+var fs = require('fs');
+
+var server = http.createServer(function(request, response) {
+    var url = request.url;
+    console.log('URL', url);
+    if (url === '/') {
+        fs.readFile('index.html', function(err, data) {
+            if (err) console.log(err);
+            else {
+                response.write(data.toString());
+                response.end();
+            }
+        })
+    } else if (url === '/page-2') {
```

Comment:
Also need to account for 'page-3' in the conditional...should be an OR conditional (`||`)